notebooks/nuextract-structure-extraction/nuextract-structure-extraction.ipynb

```diff
@@ -0,0 +1,824 @@
+{
```

Comment:
<p>explicit tokenizers conversion used in old notebooks for transition from optimum to genai, because users may have already converted models by previous notebook version that did not use openvino tokenizers and did not care about its conversion (e.g. converted when openvino tokenizers has not been integrated in optimum yet or happens dependencies mismatch between ov and tokenizers). This is new notebook, already written to use only genai that required tokenizer, so situation when user has model and does not have tokenizer never happens, please remove this code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Removed</p>